wire/carrier.go

```diff
@@ -0,0 +1,38 @@
+package wire
+
+// ProtobufCarrier is a DelegatingCarrier that uses protocol buffers as the
+// the underlying datastructur. The reason for implementing DelagatingCarrier
+// is to allow for users to serialize the underlying protocol buffers using
+// jsonpb or any other serialization forms they want.
+type ProtobufCarrier struct {
```

Comment:
I'm not reading this as carefully as I perhaps should, but am I right in understanding that this DelegatingCarrier impl is not actually used anywhere (yet)?